charts/argo-rollouts/templates/argo-rollouts-deployment.yaml

```diff
@@ -2,10 +2,19 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ .Release.Name }}
+  {{- if .Values.podAnnotations }}
+  annotations:
+  {{- range $key, $value := .Values.podAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
+  {{- end }}
   labels:
     app.kubernetes.io/component: {{ .Values.controller.component }}
     app.kubernetes.io/name: {{ .Release.Name }}
     app.kubernetes.io/part-of: {{ .Release.Name }}
+    {{- if .Values.podLabels }}
+{{- toYaml .Values.podLabels | nindent 4 }}
```

Comment:
Labels also have to be strings too right? Do we want to ensure this with the `quote` function again?

Reply:
We certainly can, I just don't see it as often.